cmd/podman/run.go

```diff
@@ -61,6 +63,19 @@ func runCmd(c *cliconfig.RunValues) error {
 		return errors.Wrapf(err, "error creating libpod runtime")
 	}
 	defer runtime.DeferredShutdown(false)
+	defer func() {
+		if c.Bool("rmi") {
+			imageName := c.InputArgs[0]
+			newImage, err := runtime.NewImageFromLocal(imageName)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "%s", errors.Wrapf(err, "failed creating image object"))
+			}
```

Comment:
This should return, since the next line is going to use an undefined newImage if the NewImageFromLOcal failed.